cmd/flightctl/main.go

```diff
@@ -13,16 +13,23 @@ import (
 	"path/filepath"
 	"regexp"
 	"strings"
+	"text/tabwriter"
 
 	api "github.com/flightctl/flightctl/api/v1alpha1"
 	"github.com/flightctl/flightctl/internal/client"
 	"github.com/flightctl/flightctl/internal/config"
 	"github.com/flightctl/flightctl/internal/crypto"
 	"github.com/spf13/cobra"
+	"github.com/thoas/go-funk"
+
 	yamlutil "k8s.io/apimachinery/pkg/util/yaml"
 	"sigs.k8s.io/yaml"
 )
 
+const (
+	YAML = "yaml"
```

Comment:
```suggestion
	yamlFormat = "yaml"
```
or something like that

Reply:
Done